src/generators/integer/index.ts

```diff
@@ -4,13 +4,13 @@ import {Chance} from 'chance';
  * Generates a random integer (could be negative!). Optionally between `lowerExclusive` and `upperExclusive`.
  */
 const createIntegerGenerator = (chance: Chance.Chance) => (
-  lowerInclusive = -100,
-  upperExclusive = 100,
+  lowerInclusive?: number,
+  upperExclusive?: number,
 ): number =>
   chance.integer({
     min: lowerInclusive,
     // chance.integer's max param is inclusive
-    max: upperExclusive - 1,
+    max: upperExclusive && upperExclusive - 1,
```

Comment:
Good Eggs best practices specify not to rely on truthiness, plus using boolean operators as ternaries is really inexpressive IMO. Plus if I'm reading this right this breaks the behavior for a max of zero, e.g. `fake.integer(-10, 0)` now will generate random ints from `-10` to `0` inclusively rather than excluding zero, because zero is falsey.

Reply:
Fixed in f9736d2